### DISCUSSION TITLE
what to do when you don't know the annotator ID?

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#210] [Q&A] [answered]
I'm importing the annotations for the vandam 5-minutes corpus, and there is no information about who the annotators were.

Shall we have a standardized name for unknown annotators? Perhaps "unknown" or "anonymous"?

And in https://childproject.readthedocs.io/en/latest/format.html#format-input-annotations -- should set be the annotator (unknown)?

**Comment by lucasgautheron** [comment, marked as answer]
I'd suggest `an1`, `an2`, `an3` etc. and so forth, which may stand both for annotator or anonymous, what do you think ?
